api/ttn/lorawan/v3/devicerepository.proto

```diff
@@ -387,11 +387,7 @@ message GetTemplateRequest {
 }
 
 message GetEndDeviceProvisioningInfoRequest {
-  // VendorID managed by the LoRa Alliance, as defined in TR005.
-  uint32 vendor_id = 1 [(validate.rules).uint32 = {gte: 1}];
-
-  // ID of the LoRaWAN end device profile assigned by the vendor.
-  uint32 vendor_profile_id = 2;
+  LoRaAllianceProfileIdentifiers lora_alliance_profile_ids = 1;
```

Comment:
Add validation that this is a required field

Reply:
Added!